SparkPost/2web2ui
# FE-1343 - Removes __func__ tests

[FE-1343](https://sparkpost.atlassian.net/browse/FE-1343)

### What Changed

- Removes all tests in `src/__func__`
- Removes unused test helpers in `src/__testHelpers__`
- Adds remaining helpers in `src/__testHelpers__` to the relevant index file
- **Note:** This might be easier to review by reviewing one commit at a time

### How To Test

- Verify unit tests still pass
- Verify we don't have a _massive_ drop in code coverage - Looks like a dip of .2% after deleting nearly 10,000 lines of tests. I would say worth it.

### To Do
